Resources/public/js/views/services/ez-sectionserversideviewservice.js

```diff
@@ -48,7 +50,89 @@ YUI.add('ez-sectionserversideviewservice', function (Y) {
                     struct.content.get('id'), update, tasks.add()
                 );
             });
-            tasks.done(Y.bind(data.afterUpdateCallback, this));
+            
+            tasks.done(
+                Y.bind(this._assignSectionCallback, this, data.sectionId, data.sectionName, e.selection, Y.bind(data.afterUpdateCallback, this))
+            );
+        },
+
+        /**
+         * Assign section callback trigerring notification and calling given callback
+         * 
+         * @method _assignSectionCallback
+         * @protected
+         * @param {String} sectionId the section id
+         * @param {String} sectionName the section name
+         * @param {Array} contents the array of contents to which section is being assgined to
+         * @param {Function} callback the callback to call when other tasks are done
+         */
+        _assignSectionCallback: function (sectionId, sectionName, contents, callback) {
+            this._assignSectionNotificationDone(sectionId, sectionName, contents);
+            callback();
+        },
+
+        /**
+         * Notification changed to *started* before assigning section to contents
+         * 
+         * @method _assignSectionNotificationStarted
+         * @protected
+         * @param {String} sectionId the section id
+         * @param {String} sectionName the section name
+         * @param {Array} contents the array of contents to which section is being assgined to
+         */
+        _assignSectionNotificationStarted: function (sectionId, sectionName, contents) {
+            var notificationIdentifier = this._getAssignSectionNotificationIdentifier('assign-section', sectionId, contents);
+
+            this.fire('notify', {
+                notification: {
+                    identifier: notificationIdentifier,
+                    text: 'Assigning the section "' + sectionName + '" to ' + contents.length + ' content' + ((contents.length>1) ? 's' : ''),
+                    state: 'started',
+                    timeout: 0
+                },
+            });
+        },
+
+        /**
+         * Notification changed to *done* after assigning section to contents
+         * 
+         * @method _assignSectionNotificationDone
+         * @protected
+         * @param {String} sectionId the section id
+         * @param {String} sectionName the section name
+         * @param {Array} contents the array of contents to which section is being assgined to
+         */
+        _assignSectionNotificationDone: function (sectionId, sectionName, contents) {
+            var notificationIdentifier = this._getAssignSectionNotificationIdentifier('assign-section', sectionId, contents);
+
+            this.fire('notify', {
+                notification: {
+                    identifier: notificationIdentifier,
+                    text: 'Section "' + sectionName + '" assigned to ' + contents.length + ' content' + ((contents.length>1) ? 's' : ''),
```

Comment:
you don't need to handle the plural here, we'll need to dig in this topic when we'll implement i18n/l10n, so the string can directly be: `'Section "' + sectionName + '" assigned to ' + contents.length + ' contents'`